app/client/components/set_table.es6

```diff
@@ -4,23 +4,28 @@ import classNames from 'classnames';
 import FontAwesome from './font_awesome.es6';
 
 const SetTable = ({ sets, selected_set, onSetClick, hideOwner, addLink }) => {
-  return (
-    <table className="table table-striped table-hover">
-      <thead>
-        <tr>
-          <th>Name</th>
-          <th>Created</th>
-          <th>Size</th>
-          { !hideOwner && <th>Owner</th>}
-        </tr>
-      </thead>
-      <tbody>
-        { sets.map((set, index) => {
-          return <SetRow set={set} selected={(set.id == selected_set)} onClick={onSetClick} key={index} hideOwner={hideOwner} addLink={addLink} />;
-        })}
-      </tbody>
-    </table>
-  )
+  if (sets.length !== 0) {
+    return (
+      <table className="table table-striped table-hover">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Created</th>
+            <th>Size</th>
+            { !hideOwner && <th>Owner</th>}
+          </tr>
+        </thead>
+        <tbody>
+          { sets.reverse().map((set, index) => {
```

Comment:
SetTable should just display what it's given. The Sets are sorted in the mapStatetoProps for the component, needs reversing there instead.